Gemfile

```
@@ -5,4 +5,5 @@ git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
 # Specify your gem's dependencies in danger-suggester.gemspec
 gemspec
 
+gem "pry"
```

```suggestion
gem 'pry'
```




<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>